.eslintrc.json

```diff
@@ -7,6 +7,7 @@
       "function-paren-newline": 0,
       "func-names": 0,
       "implicit-arrow-linebreak": 0,
-      "no-use-before-define": ["error", { "variables": false }]
+      "no-use-before-define": ["error", { "variables": false }],
+      "react/prop-types": ["error", { "ignore": ["navigation"] }]
```

Comment:
Why does navigation not need proptypes?

Reply:
There was an issue where bringing in the 'react-native' function ```navigation.navigate``` meant that I could not figure out how to test both parts of the prop. It ended up taking too long to figure out so I settled on this solution for the linter.